modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -1713,7 +1726,7 @@ sub hgvs_protein {
 
 sub hgvs_offset {
   my $self = shift;
-  return $self->{shift_hash}->{_hgvs_offset};
+  return defined($self->{shift_hash}) ? $self->{shift_hash}->{_hgvs_offset} : $self->{_hgvs_offset};
```

Comment:
A comment would be useful, why/when `_hgvs_offset` is part of `shift_hash` and when part of `self`. Thanks.